tests/data/spe1_st/tests/conftest.py

```diff
@@ -2,17 +2,17 @@
 import pytest
 
 
-@pytest.fixture(scope="session")
-def get_info():
+@pytest.fixture(scope="session", autouse=True)
+def start_storage_server():
     from ert.shared.services import Storage
 
-    with Storage.start_server() as service:
+    with Storage.connect_or_start_server() as service:
         yield service.fetch_url(), service.fetch_auth()[1]
 
 
-@pytest.fixture(scope="session")
-def requests_get(get_info):
-    base_url, token = get_info
+@pytest.fixture()
+def requests_get(start_storage_server):
```

Comment:
I might be wrong here but I think after the refactor you can remove this fixture and also the `get_experiment_dict` fixture